firefox-ios/Client/Frontend/Settings/Main/Debug/FasterInactiveTabs.swift

```diff
@@ -5,6 +5,39 @@
 import Foundation
 import Shared
 
+enum FasterInactiveTabsOption: Int {
+    case normal
+    case tenSeconds
+    case oneMinute
+    case twoMinutes
+
+    var nextOption: FasterInactiveTabsOption {
+        switch self {
+        case .normal:
+            return .tenSeconds
+        case .tenSeconds:
+            return .oneMinute
+        case .oneMinute:
+            return .twoMinutes
+        case .twoMinutes:
+            return .normal
+        }
+    }
+
+    var title: String {
+        switch self {
+        case .normal:
+            return "default"
```

Comment:
nit (non-blocking): Seems like currently, the default timeout is 14 days - it would be neat if this string reflected that as "default" is pretty vague). even better if was dynamic in the event we change the default timeout

Reply:
This is a very good idea, because even right now, the "default" just lives as a hard coded value inside Tab.swift.

I've made a ticket to refactor this and improve the debug option title further here: [FXIOS-10316](https://mozilla-hub.atlassian.net/browse/FXIOS-10316)

Just for the sake of expediency I'm going to merge and backport this now. :) Thanks for the quick review! 🙏